UNITTESTS/features/cellular/framework/AT/at_cellularstack/at_cellularstacktest.cpp

```diff
@@ -175,13 +175,13 @@ TEST_F(TestAT_CellularStack, test_AT_CellularStack_get_ip_address)
     ATHandler at(&fh1, que, 0, ",");
 
     MyStack st(at, 0, IPV6_STACK);
-    EXPECT_TRUE(0 == strlen(st.get_ip_address()));
+    EXPECT_EQ(strlen(st.get_ip_address()), 0);
 
     char table[] = "1.2.3.4.5.65.7.8.9.10.11\0";
     ATHandler_stub::ssize_value = -1;
     ATHandler_stub::bool_value = true;
     ATHandler_stub::read_string_value = table;
-    EXPECT_TRUE(NULL == st.get_ip_address());
+    EXPECT_TRUE(st.get_ip_address() == NULL);
```

Comment:
Didi you mean EXPECT_EQ(st.get_ip_address(), NULL); ?

Reply:
NULL is an exception. GTest does not allow comparision to NULL with EXPECT_EQ.

Reply:
Is the change of the operands order relevant to how the fail is reported or just a code style issue?

Reply:
Just a code style to follow other EXCEPT checks.

According to Google Test documentation:

> However, when possible, ASSERT_EQ(actual, expected) is preferred to ASSERT_TRUE(actual == expected), since it tells you actual and expected's values on failure.

So I just used the same format for all.